src/components/frame/Extensions/layouts/partials/ColonySwitcherContent/ColonySwitcherContent.tsx

```diff
@@ -81,8 +42,11 @@ const ColonySwitcherContent: FC = () => {
           <ColonySwitcherItem
```

Comment:
You can create another prop here for `colony` and pass the full colony object. Then in the `ColonySwitcherItem` you could get the displayName using `colony?.metadata?.displayName`

Reply:
@arrenv this is the kind of "dummy" component where you should pass just the needed data, so the whole `colony` object shouldn't be passed there. It will be easier to make any changes in the future if needed.